moirae/models/ecm/simulator.py

```diff
@@ -7,6 +7,7 @@
                                ECMMeasurement)
 
 
+# TODO (wardlt): Make this not part
```

Comment:
I assume there was more to this sentence? Or maybe it should be obvious and I'm just not getting it?

Reply:
No, I probably just got distracted mid-sentence 🤦🏼